ui_framework/components/tool_bar/_tool_bar_search.scss

```diff
@@ -27,12 +27,14 @@
 .kuiToolBarSearchBox {
   flex: 1 1 auto;
   position: relative;
+  font-size: $fontSize;
 }
 
   .kuiToolBarSearchBox__icon {
     position: absolute;
-    top: 6px;
-    left: 8px;
+    top: 0.5em;
+    left: 0.7em;
+    font-size: 1em;
```

Comment:
No, it was an accident. But it also doesn't have any effect on the UI) so I cleaned it up in https://github.com/elastic/kibana/pull/10358 instead.